src/main/java/features/command/CommandHandler.java

```diff
@@ -0,0 +1,247 @@
+package features.command;
+
+import features.task.DeadlineTask;
+import features.task.EventTask;
+import features.task.TaskManagement;
+import features.task.TodoTask;
+import features.task.Task;
+import utils.Utils;
+import config.Config;
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * Handles user commands related to task management.
+ */
+public class CommandHandler {
+	private TaskManagement tm;
+
+	/**
+	 * Constructs a CommandHandler with the specified TaskManagement instance.
+	 *
+	 * @param tm the TaskManagement instance used to manage tasks
+	 */
+	public CommandHandler(TaskManagement tm) {
+		this.tm = tm;
+	}
+
+	/**
+	 * Processes the given command and executes the appropriate action.
+	 *
+	 * @param cmd the command to be processed
+	 */
+	public void handleCommand(Command cmd) {
+		String command = cmd.getName();
+		try {
```

Comment:
Can check the indentation style for  try-catch statement from Java coding standard doc.